### PR TITLE
win_virtio_driver_update_by_installer: add new testcase

### PR DESCRIPTION
**Pull request by nickzhq**
win_virtio_driver_update_by_installer.py: add new test case
ID: 2102455
Signed-off-by: Houqi (Nick) Zuo [hzuo@redhat.com](mailto:hzuo@redhat.com)

### CODE REVIEW

File: provider/win_driver_installer_test.py [hunk 1]
```diff
@@ -271,3 +276,227 @@ def iozone_test(test, params, vm, images):
...
+    error_context.context("Query the status of viofs service...",
+                          test.log.info)
+    output = session.cmd_output(viofs_sc_query_cmd)
+    return output
+
+
+@error_context.context_aware
+def viofs_svc_run(test, params, session):
+    """
+    run the virtiofs service on Windows guest
```
[the remark this answers is not in the document]
Comment (nickzhq) [reply provider/win_driver_installer_test.py:394]: > If using installer to install dirvers, virtiofs service will be created by installer but the service status is stopped,we just need to start it. While for use case that the previouse drivers were installed by iso,but we tend to upgrade them by installer, then we should create/start virtiofs service by ourselves.

Here is designed for general case. At the case, we may just need the "start". 
How about we use viofs_svc_start instead viofs_svc_run at testcase?

**Comment by nickzhq** [comment]
@xiagao @zhencliu please check again, thanks

**Comment by nickzhq** [comment]
(1/1) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_update.from_old_installer.q35: STARTED
 (1/1) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_update.from_old_installer.q35: PASS (717.15 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

**Comment by zhencliu** [comment]
add @YongxueHong

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg [hunk 2]
```diff
@@ -151,6 +151,22 @@
         - driver_update:
             only all_drivers
             type = win_virtio_driver_update_by_installer
+            basic_io_test = yes
```
Comment (xiagao) [inline qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:154]: Hi, I think it's better to moving this part to the common place, such as before L43.
And only left key parameters which is  used to start the device in qemu kvm cmd line,such as "filesystems"

**Comment by nickzhq** [comment]
This case( from_old_virtio_win_iso ) have met a product bug. In other words, this case should be failed.
Hello @xiagao please check again, thanks!

**Comment by xiagao** [comment]
> This case( from_old_virtio_win_iso ) have met a product bug. In other words, this case should be failed. Hello @xiagao please check again, thanks!

It's true. The previous virtio-win rpm pkg has an issue that can't stop virtiofs service.

**Pull request closed without merging**